VideoLocker/src/main/java/org/edx/mobile/course/CourseService.java

```diff
@@ -7,9 +7,39 @@
 import retrofit.http.Query;
 
 public interface CourseService {
+    /**
+     * @param username (optional):
+     *                 The username of the specified user whose visible courses we
+     *                 want to see. The username is not required only if the API is
+     *                 requested by an Anonymous user.
+     * @param org      (optional):
```

Comment:
Is it necessary to include parameters that are not used by the mobile app?
For example, there's currently no use case for the `org` parameter for the mobile app, so why include it in our code?